apps/voucherify/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "voucherify-contentful-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "cross-env BROWSER=none react-scripts start",
+    "postinstall": "npm run build",
+    "build": "react-scripts build",
+    "install-ci": "npm ci",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "upload": "contentful-app-scripts upload --bundle-dir ./build",
+    "upload-ci": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id $CONTENTFUL_ORG_ID --definition-id $CONTENTFUL_APP_DEF_ID --token $CONTENTFUL_ACCESS_TOKEN",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 6xt1AyXyw6aSnn5JDXaFin --token ${CONTENTFUL_CMA_TOKEN}",
```

Comment:
Just verifying that `6xt1AyXyw6aSnn5JDXaFin` is the app id for the Voucherify custom app that you have in your org?

Reply:
I made a mistake, this id is `qLqU3Zs2d5awizzTcZ1mK`. :) 
This is the id of the Custom App that we created within the organization "Rspective" and space "Voucherify". Within this organization and space we built and tested our custom app. Please keep in mind that this app (its URL is: `https://cms-integrations.voucherify.io/frontend/contentful`) is slightly different from what is included in the PR, especially when it comes to the appearance and operation of the Configuration Screen.
<img width="1260" alt="image" src="https://github.com/contentful/marketplace-partner-apps/assets/117282008/43fe80c3-7b7b-45c5-b296-4c5d8cb7a29b">

Reply:
@weronika-kurczyna Sorry, one more change here. Could you update the app id in the deploy script to this `3HFf3DxTPYMh2L8214oW1s`? As mentioned in our email, we are going to use a fresh app definition for this.

Reply:
@weronika-kurczyna Just FYI - I made this change just to make it easier to get this PR done for release tomorrow!